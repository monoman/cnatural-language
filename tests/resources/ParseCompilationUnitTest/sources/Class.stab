class Point : Move {
	int x, y;
	void move(int dx, int dy) { x += dx; y += dy; }
}
