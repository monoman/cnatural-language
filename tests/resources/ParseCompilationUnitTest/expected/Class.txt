class Point : Move {
    int x, y, c♮;
    void translação(int dx, int dy) {
        x += dx;
        y += dy;
    }
}
